tasks/2_convert-to-f/convert-to-f.js

```diff
@@ -0,0 +1,6 @@
+export function convertToF(celsius) {
+  var fahrenheit;
+  fahrenheit = celsius * 9 / 5;
```

Comment:
It would be better to combine it in 1 line and use const instead of var

Reply:
> It would be better to combine it in 1 line and use const instead of var

Have changed to const instead of var and combined it in 1 line